content/en/docs/concepts/containers/container-lifecycle-hooks.md

```diff
@@ -37,9 +37,9 @@ No parameters are passed to the handler.
 `PreStop`
 
 This hook is called immediately before a container is terminated due to an API request or management
-event such as a liveness/startup probe failure, preemption, resource contention and others. A call
+events such as a liveness/startup probe failure, preemption, resource contention, and others. A call
 to the `PreStop` hook fails if the container is already in a terminated or completed state and the
-hook must complete before the TERM signal to stop the container can be sent. The Pod's termination
+hook must be complete before the TERM signal to stop the container can be sent. The Pod's termination
```

Comment:
```suggestion
hook must complete before the TERM signal to stop the container can be sent. The Pod's termination
```
You can either leave it as it was or if you want to introduce `be` I believe the line would be

```suggestion
hook must be completed before the TERM signal to stop the container can be sent. The Pod's termination
```